darwin/options.py

```diff
@@ -416,9 +420,11 @@ def cpu_default_types(input: Any) -> Optional[int]:  # type: ignore
         )
         parser_convert.add_argument(
             "--legacy",
-            action="store_true",
-            default=False,
-            help="Import annotation files using legacy process (isotropic transformation).",
+            type=lambda x: (str(x).lower() == "true") if x is not None else None,
+            nargs="?",
+            const=True,
+            default=None,
+            help="Import annotation files using legacy process (isotropic transformation). Pass 'true' to enable, or omit for None.",
```

Comment:
Think this should be aligned w the console logs that say

`"The `legacy` flag is now non-functional and will be deprecated soon. The annotation import process now automatically detects if legacy annotation scaling is required.",`